functions/_common/grapherRenderer.ts

```diff
@@ -210,15 +209,38 @@ async function fetchAndRenderGrapherToSvg(
 
     if (fetchResponse.status !== 200) {
         console.log("Failed to fetch grapher config", fetchResponse.status)
-        return null
+        return {
+            grapherConfig: null,
+            status: fetchResponse.status,
+            etag: fetchResponse.headers.get("etag"),
+        }
     }
```

Comment:
I brought this up in the other PR already:
We definitely want to be handling `304 Not Modified`.

Looking at this code, it actually might be handling this all fine already if the 304 status is correctly forwarded to the client. But the `console.log` message is misleading in that case :)